### DISCUSSION TITLE
Do you plan to launch an official Android client?

### DISCUSSION THREAD

**Discussion by zhuerhao** [navidrome/navidrome#2439] [Ideas]
Do you plan to launch an official Android client? None of the third-party clients recommended on the Internet can display lyrics (including DSub, Subtracks, substreamer, Ultrasonic and Audinaut)

**Comment by metalheim** [comment]
Symfonium can display lyrics. Its paid, closed source software, but well worth the price imho

**Comment by dweymouth** [comment]
It looks like the newly released Tempo client supports lyrics: https://www.reddit.com/r/navidrome/comments/14iisto/sharing_tempo_a_lightweight_opensource_music/

**Comment by deluan** [comment]
There's no current plans to release an "official" client, but there are plans to improve the API, which would allow better functionalities in the future.

By the way, DSub can display lyrics, but it is not the best experience:
![Screenshot_20230710-222420](https://github.com/navidrome/navidrome/assets/331353/ad2fdb32-0f14-4124-964f-32da2a8cf032)
